### ISSUE TITLE
请问如何进行无坐标的训练？

### ISSUE THREAD

**Issue by Airwithcode** [shannanyinxiang/SPTS#3]
需要修改哪一块的代码

**Comment by shannanyinxiang** [comment]
修改dataset/collate_fn.py中的SeqConstructor类，使其在构建序列时仅包括文本序列。

具体地，可以将dataset/collate_fn.py中43行的`pt_label = torch.cat([center_pts, recog_label], dim=-1)`修改为`pt_label = recog_label`。

**Comment by Airwithcode** [comment]
感谢您的回答，训练完成后，推理没有结果，请问需要修改哪部分代码

**Comment by shannanyinxiang** [comment]
修改engine/val.py中的decode_pred_seq函数，让它只解码文本。还需要修改model/transformer.py的71行到79行。
